tsconfig.json

```diff
@@ -8,5 +8,5 @@
   "include": [
     "src/**/*",
     "test/**/*"
-  ]
+, "template"  ]
```

Comment:
including this in the tsconfig means the code will be transpiled. Was this the plan here?